src/hooks/Tokens.ts

```diff
@@ -477,3 +477,53 @@ export function useHederaTokenAssociated(token: Token | undefined): {
     };
   }, [chainId, tokenAddress, account, loading, isLoading, isAssociated]);
 }
+
+export function useHederaAddressAssociated(address: string | undefined): {
```

Comment:
why do we need extra hook `useHederaAddressAssociated`, i think we can update `useHederaTokenAssociated` to accept token address instead of token instance and we can pass something like symbol or name that will be useful to show toast notification. what do you think ?